package.json

```diff
@@ -4,8 +4,11 @@
   "version": "2.6.2",
   "main": "./core/lib/patternlab.js",
   "dependencies": {
+    "async": "^2.1.2",
+    "dive": "^0.5.0",
     "diveSync": "^0.3.0",
```

Comment:
#389  exists to attempt to merge a couple dependencies. do you see us still being able to do so with the addition of the true async dive?